pipeline_live/data/sources/alpaca.py

```diff
@@ -6,10 +6,11 @@
 
 
 def list_symbols():
-    return [
-        a.symbol for a in tradeapi.REST().list_assets()
-        if a.tradable and a.status == 'active'
-    ]
+    with tradeapi.REST() as api:
```

Comment:
Hi, 
don't need the `with` statement.. just do: `api = tradeapi.REST()` and use `api` in the list comprehension

Reply:
Thanks I've updated the PR